lib/steps/setupRedirects.js

```diff
@@ -32,53 +33,56 @@ const setupRedirects = (publishPath) => {
     ...require("../pages/withoutProps/redirects"),
   ];
 
+  // Add _redirect section heading
+  redirects.push("# Next-on-Netlify Redirects");
+
+  const staticRedirects = nextRedirects.filter(
+    ({ route }) => !isDynamicRoute(route)
+  );
+  const dynamicRedirects = nextRedirects.filter(({ route }) =>
+    isDynamicRoute(route)
+  );
+
   // Add next/image redirect to our image function
-  nextRedirects.push({
+  dynamicRedirects.push({
     route: "/_next/image*  url=:url w=:width q=:quality",
     target: `/.netlify/functions/${NEXT_IMAGE_FUNCTION_NAME}?url=:url&w=:width&q=:quality`,
   });
 
-  // Add _redirect section heading
-  redirects.push("# Next-on-Netlify Redirects");
-
-  // Sort routes: More-specific routes (e.g., static routing) precede
-  // less-specific routes (e.g., catch-all)
-  const sortedRoutes = getSortedRoutes(nextRedirects.map(({ route }) => route));
-
-  // There may be several redirects with the same route but different targets
-  const wasRedirectAdded = (redirect) => {
-    return redirects.find((addedRedirect) => {
-      const [route, target] = addedRedirect.split("  ");
-      return redirect.route === route && redirect.target === target;
-    });
+  // Sort routes: More-specific routes precede less-specific routes (e.g., catch-all)
+  const getSortedRedirects = (redirects) => {
+    const sortedRoutes = getSortedRoutes(redirects.map(({ route }) => route));
+    return redirects.sort(
+      (a, b) => sortedRoutes.indexOf(a.route) - sortedRoutes.indexOf(b.route)
+    );
```

Comment:
Thoughts on changing `helpers/getSortedRoutes` into `helpers/getSortedRedirects` and abstracting this logic into the helper?